### PR TITLE
fix:[CDS-76711]: Add new index

### PR DESCRIPTION
**Pull request by Fernando-Dourado**
## Describe your changes
- Added a new index using accountId, createdBy.email, and createdAt fields.
- Sharing an image with execution status using the newer index.
- If needed I put the prod execution status to the ticket.

Screenshot 2023-08-10 at 09 15 34

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/51567)

### CODE REVIEW

**Comment by Fernando-Dourado** [comment]
trigger uts

**Comment by Fernando-Dourado** [comment]
trigger messagecheck

**Comment by Fernando-Dourado** [comment]
trigger uts

**Comment by Fernando-Dourado** [comment]
trigger uts_JDK11

File: 400-rest/src/main/java/software/wings/beans/WorkflowExecution.java [hunk 1]
```diff
@@ -130,6 +131,8 @@ public class WorkflowExecution implements PersistentRegularIterable, AccountData
       "accountId_appId_serviceIds_createdAt_cdPageCandidate_status";
   public static String ACCOUNTID_APPID_ENVIDS_CREATEDAT_CDPAGECANDIDATE_STATUS =
       "accountId_appId_envIds_createdAt_cdPageCandidate_status";
+  public static final String ACCOUNTID_CREATEDBY_EMAIL_CREATEDAT = "accountId_createdBy-email_createdAt";
+  private static String createdBy_email = "createdBy.email";
```
Comment (lasdox) [inline 400-rest/src/main/java/software/wings/beans/WorkflowExecution.java:135]: there is a UtilityClass at this same file that overrides WorkflowExecutionKeys, change it to there please

**Comment by Fernando-Dourado** [comment]
trigger uts

**Review by lasdox — APPROVED** [review]

**Review by rafaelmart1ns — APPROVED** [review]

**Comment by Fernando-Dourado** [comment]
trigger review

**Pull request merged**